src/components/cluster/Cluster.tsx

```diff
@@ -212,7 +212,7 @@ export default class ClusterList extends Component<ClusterListProps, any> {
                 'Clusters' +
                 (this.props.serverMode === SERVER_MODE.EA_ONLY || window._env_.K8S_CLIENT ? '' : ' and Environments')
             return (
-                <section className="mt-16 mb-16 ml-20 mr-20 global-configuration__component flex-1">
+                <section className="global-configuration__component flex-1">
```

Comment:
Classname global-configuration__component is not making any sense in this component. Please rename it to the respective component